src/viam/services/motion/client.py

```diff
@@ -188,6 +223,23 @@ async def move_on_map(
 
         You can monitor the progress of the ``move_on_map()`` call by querying ``get_plan()`` and ``list_plan_statuses()``.
 
+        ::
+
+            motion = MotionClient.from_robot(robot=robot, name="builtin")
+
+            # Get the ResourceNames of the base and SLAM service
```

Comment:
(minor) should we specify "base _component_" since we're specifying SLAM is a a `service`?